ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -755,29 +758,44 @@ def test017_update_shipment_date(self, save):
         LIMS-4779
```

Comment:
why you update This test case in this PR, while travis file contains only LIMS-3406

Reply:
Done